components/results_table/commons/results_table_award.lua

```diff
@@ -63,9 +63,9 @@ function AwardsTable:buildRow(placement)
 	row:tag('td'):css('text-align', 'left'):wikitext(placement.extradata.award)
 
 	if self.config.playerResultsOfTeam or self.config.opponentType ~= Opponent.team then
-		row:tag('td'):css('text-align', 'right'):attr('data-sort-value', placement.opponentname):node(self:opponentDisplay(
+		row:tag('td'):css('text-align', 'left'):attr('data-sort-value', placement.opponentname):node(self:opponentDisplay(
```

Comment:
check if the `:css('text-align', 'left')` can just be kicked

Reply:
Its centered by default, so required I would say